### DISCUSSION TITLE
alt podman

### DISCUSSION THREAD

**Discussion by dyakoff** [NiceNode/nice-node#675] [Ideas]
Hello. Cool software you saw. But podman is just pain and suffering. Only because of him I had to stop testing. Make an alternative to choose from Podman or Docker. Yes, I gave the docker crutches, but it worked every other time. I don't know if it's a bug or not. Docker is more familiar and much more stable, it seems to me. Moreover, in the code I saw provisions for docker, or docker was previously used by you, then why did you refuse? Thank you for your attention.

**Comment by jgresham** [comment]
I have been thinking to add a Docker option for linux. Are you on Linux? Docker on Mac and Windows is much more complicated. @dyakoff

**Reply by dyakoff** [reply]
Yes. Linux.
